tests/aws/services/dynamodb/test_dynamodb.py

```diff
@@ -218,7 +218,7 @@ def test_stream_spec_and_region_replacement(self, aws_client):
         table = aws_client.dynamodb.describe_table(TableName=table_name)["Table"]
 
         # assert ARN formats
-        expected_arn_prefix = "arn:aws:dynamodb:" + aws_stack.get_local_region()
+        expected_arn_prefix = "arn:aws:dynamodb:" + TEST_AWS_REGION_NAME
```

Comment:
Ensures expected value is correctly constructed.

Fixes this test: test_stream_spec_and_region_replacement